### PR TITLE
Add hubiC to backup

### PR DESCRIPTION
**Pull request by marsuboss**

### CODE REVIEW

File: _data/backup.yml [hunk 1]
```diff
@@ -88,6 +88,12 @@ websites:
     hardware: Yes
     doc: http://www.google.com/intl/en-US/landing/2step/features.html
 
+  - name: hubiC
+    url: https://hubic.com//
```
Comment (Carlgo11) [inline _data/backup.yml]: change the url to `https://hubic.com/`

**Comment by mxxcon** [comment]
Looks like this is a service from OVH and their 2fa status is a mess. :confounded:

**Comment by Carlgo11** [comment]
The size of the logo is too large. Here's a compressed version:
hubic

**Comment by marsuboss** [comment]
@Carlgo11 Done

**Comment by fpigerre** [comment]
Thanks for the pull request @marsuboss! :+1:

**Pull request merged**